optuna/samplers/_tpe/sampler.py

```diff
@@ -94,6 +98,30 @@ def objective(trial):
             study = optuna.create_study(sampler=TPESampler())
             study.optimize(objective, n_trials=10)
 
+    .. note::
+        For `v2.9.0 <https://github.com/optuna/optuna/releases/tag/v2.9.0>`_ or later,
+        :class:`~optuna.samplers.MOTPESampler` is deprecated and
+        :class:`~optuna.samplers.TPESampler` should be used instead. The following code shows
```

Comment:
I feel including deprecation information of `MOTPESampler` in the doc of `TPESampler` is not necessary and rather misleading, but what do you think?

Reply:
The background of this PR was actually when I looked for the usage of MOTPE, `MOTPESampler` showed up on the google search and I was wondering how I should use `multivariate` there.

I was wondering if it has not just been supported yet because `TPESampler`, which has the `multivariate` feature, does not mention anything about how it should be applied to multi-objective optimization.

So I thought that it would be nice to have a note saying "you can use this sampler for multi-objective optimization in this way!" ALTHOUGH I now agree that it is not necessary to have this note given the fact that we deprecated `optuna/multi_objective` from the documentation.

Reply:
I changed here accordingly.